README.md

```diff
@@ -127,6 +127,7 @@ MyParsingClass {
 ## Future Enhancements
 
 - Generate code so that member variables only need to be package local
+- Add an option to absorb parsing errors rather than re-throwing them
```

Comment:
I think this would be a good option. We can discuss it, but it could be done on a class by class basis or globally. Basically if we hit an IOException we would check if the user wants to be notified by rethrowing it, otherwise we would swallow it and proceed with parsing.